File: features/transaction/transactiondata/src/main/java/com/tokopedia/transactiondata/entity/response/shippingaddressform/Dropshiper.java
package com.tokopedia.transactiondata.entity.response.shippingaddressform;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Irfan Khoirul on 24/09/18.
 */

public class Dropshiper {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("telp_no")
    @Expose
    private String telpNo;

    public Dropshiper() {
    }

    public String getName() {
        return name;
    }

    public String getTelpNo() {
        return telpNo;
    }
}

File: features/transaction/checkout/src/main/java/com/tokopedia/checkout/data/model/response/shipmentaddressform/Dropshiper.kt
package com.tokopedia.checkout.data.model.response.shipmentaddressform

import com.google.gson.annotations.SerializedName

data class Dropshiper(
        @SerializedName("name")
        val name: String = "",
        @SerializedName("telp_no")
        val telpNo: String = ""
)